packages/icons/icons.json

```diff
@@ -4051,5 +4051,19 @@
         "tags": "microphone, mic, speaker",
         "group": "media",
         "codepoint": 62275
-    }
+    },
+    {
+        "displayName": "Geotime",
+        "iconName": "geotime",
+        "tags": "geo, time",
+        "group": "interface",
+        "codepoint": 62276
+      },
```

Comment:
are you using [EditorConfig](https://editorconfig.org/)? that should help with whitespace issues like this.

Reply:
What whitespace issue? I can't really see any specifics here?